### ISSUE TITLE
[Feature] Category or Main Theme tags

### ISSUE THREAD

**Issue by MammutReal** [stashapp/stash#5314]
Hi!

I tried to look for this but I didn't see similar request.

There are days when I'm in the mood of, let's say, some doggy. But it's many times only a minute temporary position in the middle of an hour long scene, so it's not the main tag of the scene.
Of course, tags should show everything from a scene, so you can find a scene later, but a scene can contains many things which are only details and not a theme.
I'd like to filter to the theme, too.

I'd like a separate tag field at scenes, groups, galleries and images, named Category or Main theme or similar.

Another solution would be if we could attach an "importance" value to the assigned tags, but i think that would be more complex.

Surely, we can make markers, but you can't see the duration from that either, and I think it would be better if the scene info itself would show the main tags and "detail" tags right away anyway.

Thanks in advance!

**Comment by stg-annon** [comment]
There are actually a few open issues related to this

you might be able to use a few workarounds to fit your use case, you could focus on studios that focus on a specific type/style of content as something to filter by. You could create a group that is essentially a theme that scenes could be placed into. you could create another custom tag that is used for themes that you could then use to filter by i.e. `Theme - Doggy` it would be more of a manual process but I have seen scraping sources that have similar tags


>Another solution would be if we could attach an "importance" value to the assigned tags, but i think that would be more complex

see issue #2973


>Surely, we can make markers, but you can't see the duration from that either

that is another issue that is in a PR right now #5311 so we should eventually have a duration

with the introduction of marker duration we may be able to make a more weighted filter that focuses on the total duration of a tag on a scene via markers which I believe would solve both issues

**Comment by MammutReal** [comment]
> You could create a group that is essentially a theme that scenes could be placed into. you could create another custom tag that is used for themes that you could then use to filter by i.e. `Theme - Doggy` it would be more of a manual process but I have seen scraping sources that have similar tags

Hm, yes, but basically most of the tags would be duplicated with time this way, so I think this can be only a temporary solution for some tags but not for all of them. 

> see issue #2973

Thanks, I didn't see that. Although I think this way the tagging would be a more complex process (and it's already a lengthy process as it is now), so I'm not sure if it would be the best solution. The option itself would be useful anyway, of course.

> that is another issue that is in a PR right now #5311 so we should eventually have a duration

> with the introduction of marker duration we may be able to make a more weighted filter that focuses on the total duration of a tag on a scene via markers which I believe would solve both issues

I guess it connects also to the many smaller scenes in a bigger video problem. But sure, this would be the most precise as you can jump there right away. 

Thanks!

**Comment by tobylibo** [comment]
Yes, I need this feature, too. But I think the developers would prefer to let us use groups and tags instead of categories. So I can only accept this fact. Of course, I'm still looking forward to adding this feature one day.

**Comment by MammutReal** [comment]
Well, the groups are for complete movies (splitted scene files) and series, no? It would be a little chaotic if we'd start to use them as grouping for themes. 
I think the categories would be a more transparent user experience, and it's just one more tag-like field, and one more table, like the scenes_tags, with scene_id and tag_id as we'd use the same existing tags for this.
I hope they'll reconsider. The marker's duration would be a solution, too (although I found out that I don't really like to use them because of their thumbs, the scene thumbs are way better).

**Comment by MammutReal** [comment]
> Yes, I need this feature, too. But I think the developers would prefer to let us use groups and tags instead of categories. So I can only accept this fact. Of course, I'm still looking forward to adding this feature one day.

Another problem with Groups: I can't use them for Galleries.

**Comment by MammutReal** [comment]
I started to build a Theme-system in the Groups, but frankly, it just duplicates the Tags, because too many tags can be theme as well. I still think that a Tag-like Category field which uses the same, already built tag-hierarchy would be faster.
